daemon/volumes_unix.go

```diff
@@ -48,7 +48,7 @@ func (container *Container) setupMounts() ([]execdriver.Mount, error) {
 		}
 		if !container.trySetNetworkMount(m.Destination, path) {
 			mounts = append(mounts, execdriver.Mount{
-				Source:      path, // Note that for Ceph volumes, this will be the mapped device (e.g. /dev/rbd0), and for NFS shares, it will be the share URI (e.g. 1.2.3.4://foo)
+				Source:      path, // Note that for Ceph volumes, this will be the mapped device (e.g. /dev/rbd0), and for NFS shares, it will be the mounted host folder
```

Comment:
I suggest using "directory" (or "dir" if you need to abbreviate) instead of "folder" everywhere, since it's more Unix-style.